backends-velox/src/test/scala/org/apache/gluten/execution/VeloxAggregateFunctionsSuite.scala

```diff
@@ -1193,6 +1193,28 @@ class VeloxAggregateFunctionsDefaultSuite extends VeloxAggregateFunctionsSuite {
       }
     }
   }
+
+  test("aggregate on join keys can set ignoreNullKeys") {
+    val s =
+      """
+        |select count(1) from
+        |  (select l_orderkey, max(l_partkey) from lineitem group by l_orderkey) a
```

Comment:
What if agg offload but join fallback?

Reply:
it doesn't matter.  if the pattern is 
```
 agg(k1)       t
    \               /
     \            /
    join(k1 = k2) 
```

then we can safely ignore the null values in k1 when doing aggregation, no matter whether `join` is offloaded of not.